### PR TITLE
Added missing spotbugs target in Makefile

### PR DESCRIPTION
**Pull request by ppatierno**
Trivial PR to add missing Makefile target for spotbugs which will be needed in the Azure pipeline.

### CODE REVIEW

**Review by scholzj — APPROVED** [review]

File: Makefile.maven [hunk 1]
```diff
@@ -24,4 +24,8 @@ java_install:
 .PHONY: java_clean
 java_clean:
 	echo "Cleaning Maven build ..."
-	mvn clean
\ No newline at end of file
+	mvn clean
+
+.PHONY: spotbugs
+spotbugs: java_compile
```
Comment (scholzj) [inline Makefile.maven:30]: Do you really want to run `java_compile` before it? That seems like it would cause a lot of time waste in the CI.

Comment (ppatierno) [reply Makefile.maven:30]: But spotbugs needs the new code compiled in order to spot ... bugs :-)
This is what we need when developing locally, or?

Comment (scholzj) [reply Makefile.maven:30]: But it means you will recompile everything in CIs if you do it like this. We do not do that for example in operators.

Comment (ppatierno) [reply Makefile.maven]: Agree. It is useful for local development when you could forget to recompile while fixing spotbugs :-) But doesn't make sense in CI. I removed it.

**Review by antonio-pedro99 — APPROVED** [review]

**Review by kyguy — APPROVED** [review]

**Pull request merged**